src/test/java/net/minecraftforge/debug/entity/player/TradeWithVillagerEventTest.java

```diff
@@ -0,0 +1,28 @@
+package net.minecraftforge.debug.entity.player;
+
+import net.minecraftforge.event.entity.player.TradeWithVillagerEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * This tests for {@link TradeWithVillagerEvent} and fires when
+ * the player completes a trade with either a Villager or WanderingTrader(must inherit from AbstractVillager).
+ * This test shows the player name involved along with the villager name and what the result of the trade is(Count and Item).
+ */
+@Mod("trade_with_villager_event_test")
+@Mod.EventBusSubscriber()
+public class TradeWithVillagerEventTest
+{
+
+    private static final boolean ENABLE = false;
+    private static final Logger LOGGER = LogManager.getLogger(TradeWithVillagerEventTest.class);
+
+    @SubscribeEvent
+    public static void onPlayerVillagerTrade(TradeWithVillagerEvent event)
+    {
+        if (!ENABLE) return;
+        LOGGER.info("Player {} traded with villager {} and exchanged for {} {}.", event.getEntity().getName().getString(), event.getAbstractVillager().getName().getString(), event.getMerchantOffer().getResult().getCount(), event.getMerchantOffer().getResult().getDisplayName().getString());
```

Comment:
Please split this line so it is not unreasonably long. Prefer to keep it under the length of that third and longest line in your class javadoc.